modules/@apostrophecms/schema/ui/apos/components/AposInputArray.vue

```diff
@@ -269,7 +268,7 @@ export default {
 
             return {
               name: 'duplicate',
-              message: `duplicate ${this.$t(uniqueFieldSchema.label) || uniqueFieldName}`
+              message: `${this.$t('apostrophe:duplicateError')} ${this.$t(uniqueFieldSchema.label) || uniqueFieldName}`
```

Comment:
I think you can do `message: { key: 'apostrophe:duplicateError', name: $this.$t(uniqueFieldSchema.label) }` but if that's not a thing then this works OK for now

Reply:
`key` works in `message` but `name` has to be outside to name correctly the error for the css class. I let it like that.